tutorials/plot_artifacts_correction_filtering.py

```diff
@@ -25,13 +25,11 @@
 
 tmin, tmax = 0, 20  # use the first 60s of data
```

Comment:
`0, 60` (as in `examples/time_frequency/plot_compute_raw_data_spectrum.py`)?

Or `# use the first 20s of data`